src/pages/docs/administration/upgrading/guide/automate-upgrades.mdx

```diff
@@ -71,7 +71,7 @@ if ($upgradeVersion -eq $currentVersion) {
 # Download the installer
 $msiFilename = "Octopus.$upgradeVersion-x64.msi"
 Write-Host "Downloading $msiFilename"
-Start-BitsTransfer -source "https://download.octopusdeploy.com/octopus/$msiFilename" -Destination "$downloadDirectory\$msiFilename"
+Start-BitsTransfer -source $versionInfo.DownloadUrl -Destination "$downloadDirectory\$msiFilename"
```

Comment:
`Start-BitsTransfer` is a bit old and busted isn't it? Or is this a max compatibility cmdlet?